### DISCUSSION TITLE
Can we show selection border during the editing of IText ?

### DISCUSSION THREAD

**Discussion by meetgoyani007** [fabricjs/fabric.js#7581] [Q&A]
Hello,

It is possible to customize the selection border color when IText is in edit mode or it is possible to move objects while editing the text.

**Comment by asturur** [comment]
IText isn't movable while in edit mode.
I wrote some code to have it movable from an added padding, but is not a great feature i wish.
What we can check is that if we can leave the controls up ( optionally ) and you can write a custom control to drag the object.
